cs/src/core/Index/FASTER/FASTER.cs

```diff
@@ -186,67 +186,67 @@ public FasterKV(long size, Functions functions, LogSettings logSettings, Checkpo
             sectorSize = (int)logSettings.LogDevice.SectorSize;
             Initialize(size, sectorSize);
 
-            _systemState = default(SystemState);
+            _systemState = default;
             _systemState.phase = Phase.REST;
             _systemState.version = 1;
             _checkpointType = CheckpointType.HYBRID_LOG_ONLY;
         }
 
         /// <summary>
-        /// Take full checkpoint
+        /// Initiate full checkpoint
         /// </summary>
-        /// <param name="token"></param>
-        /// <returns></returns>
+        /// <param name="token">Checkpoint token</param>
+        /// <returns>Whether we could initiate the checkpoint</returns>
```

Comment:
nit: maybe it's helpful to also provide why a checkpoint might fail to be initiated here